tiledb/sm/query/writer.cc

```diff
@@ -538,6 +544,9 @@ Status Writer::check_coord_oob() const {
   auto dim_num = array_schema_->dim_num();
   auto domain = (T*)array_schema_->domain()->domain();
 
+  if (coords_num < 2)
```

Comment:
Can't you still have a single OOB coord?

Reply:
Yeap, I'll fix.